### ISSUE TITLE
Trajectories broken in esmini 2.18+

### ISSUE THREAD

**Issue by johschmitz** [esmini/esmini#203]
I just noticed that my trajectories are not working anymore in esmini. They are working fine up to the 2.17.6 release.
Here an example:
[example.zip](https://github.com/esmini/esmini/files/7799168/example.zip)

**Comment by eknabevcc** [comment]
Thanks for reporting this. 
The implementation of LaneChange was recently modified to be more flexible, e.g. maintain route. A unforeseen side effect was that any attached trajectory was lost.
It's now fixed and included in release [2.19.0](https://github.com/esmini/esmini/releases/tag/v2.19.0).

However this is an interesting case. We have three private actions in the Init section:
- TeleportAction 
- LaneChangeAction
- FollowTrajectoryAction

of which both LaneChange and FollowTrajectory operates on lateral domain (see [this table](https://www.asam.net/index.php?eID=dumpFile&t=f&f=4596&token=55bca7d8439f2bae072c4dffd1ee544a6d76b786#_action_tables)). According to rule of "conflicting actions" (see section [4.3. ManeuverGroups, Maneuvers, Events, and Actions](https://www.asam.net/index.php?eID=dumpFile&t=f&f=4596&token=55bca7d8439f2bae072c4dffd1ee544a6d76b786#_maneuvergroups_maneuvers_events_and_actions) sub-section "Conflicting actions") the latter should be overridden. I.e. LaneChange and FollowTrajectory can not happen simultaneously. (BTW, latter is interpreted as the old/ongoing one. The newest wins)
HOWEVER, we're now in the Init section and in esmini conflicts are only evaluated for events, hence the actions in Init sections goes "under the radar"... for good and bad :)

**Comment by johschmitz** [comment]
I think it is good that it is this way round and the trajectory gets priority in this scenario.

**Comment by eknabevcc** [comment]
Yes I agree. In normal case (in case of events), the latest wins. 
However in your scenario both will happen in parallel (the lane change is very quick so I hardly noted it).

**Comment by johschmitz** [comment]
Okay maybe I can skip the initial lane change when there's already a trajectory attached.

**Comment by eknabevcc** [comment]
Yes, that's probably good if possible.
I don't think it's necessary for esmini. 
But removing the laneChange (or perform it before or after the trajectory action - but this kind of relations implies moving the actions to events in a Story) would make the scenario more "correct"/harmonizing with the standard.
